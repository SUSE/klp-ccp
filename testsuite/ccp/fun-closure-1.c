void pu_f(void);

void pu_g(void)
{
	pu_f();
}

void pu_f(void)
{
	pu_g();
}
