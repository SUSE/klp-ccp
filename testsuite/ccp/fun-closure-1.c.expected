/* klp-ccp: from testsuite/ccp/fun-closure-1.c */
void klpp_pu_f(void);

void klpp_pu_g(void)
{
	klpp_pu_f();
}

void klpp_pu_f(void)
{
	klpp_pu_g();
}
